CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 Not yet released; provisionally v2.0.0 (may change).
 
+
+### OmitInclusionProof Option
+Allow map clients to forgo fetching inclusion proofs.
```

Comment:
How often will this be set? Is it better to invert it?

Reply:
Or you could have different methods rather than parameters that drastically effect performance.

Reply:
By constructing the option in this way, existing methods that don't set this option will keep their existing behavior.  That said, double negatives are admittedly an anti pattern.

Reply:
We expect this flag to not be set the vast majority of the time.

Reply:
I'll make this a new API endpoint